CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.5.1](https://github.com/ASFHyP3/hyp3-sdk/compare/v0.5.0...v0.5.1)
+## [0.6.0](https://github.com/ASFHyP3/hyp3-sdk/compare/v0.5.0...v0.6.0)
+
+### Changed
+- HyP3 prepare and submit methods now include processing options as named parameters
```

Comment:
might list the four affected functions here, since autorift is unchanged
`prepare_rtc_job`, `submit_rtc_job`, `submit_insar_job`, prepare_insar_job`